package.json

```diff
@@ -63,6 +64,12 @@
     "rimraf": "^3.0.0",
     "uint8arrays": "^2.0.5"
   },
+  "eslintConfig": {
+    "extends": "ipfs",
+    "ignorePatterns": [
+      "!.aegir.js"
```

Comment:
Why are we not linting this file?

Reply:
this actually tell the linter to **lint** the file, double negative lol

Reply:
eslint ignores dot files by default we need to tell the linter to not ignore .aegir.js inside the ignore config 🤷🏻‍♂️